model/src/cfd.rs

```diff
@@ -1535,6 +1535,8 @@ impl Cfd {
         if !hours_left.is_positive() {
             tracing::warn!("Rolling over a contract that can be settled non-collaboratively");
 
+            // TODO: This is an incorrect fallback. Should be changed to the actual hours we
+            // have  to charge or we have to disallow using events older than 24h
```

Comment:
can we raise an issue so we won't forget this? (if it's not yet tracked)

Reply:
I adapted https://github.com/itchysats/itchysats/issues/2282 and added this change there. Removed `TODO`.